### PR TITLE
ai-alina

### PR DESCRIPTION
**Pull request by alina-jacob**
Closes #https://github.com/carbon-design-system/carbon-website/issues/4036 and https://github.com/carbon-design-system/carbon-website/issues/4039

#### Changelog

**New**

Added in the Usage tab
- Inline notification
- AI Presence Section
- AI Revert Section

Added in the Style tab
- AI Styling table
- AI Default and Fluid states for respective components

### CODE REVIEW

File: src/pages/components/text-input/style.mdx [hunk 1]
```diff
@@ -326,6 +327,44 @@ These sizes apply only to the default style of text input.
...
+the default variant.
+
+[Learn more about the AI style elements]()
+
+| Element         | Property         | Token / Size      |
+| --------------- | ---------------- | ----------------- |
+| Linear gradient | start            | $ai-aura-start-sm |
+|                 | stop             | $ai-aura-stop     |
+| Field           | border-bottom    | $ai-border-strong |
+|                 | background color | $field\*          |
```
Comment (aagonzales) [inline src/pages/components/text-input/style.mdx:343]: The tokens should be in the code snippets. I didn't have this in my spec, my bad.
```suggestion
| Linear gradient | start            | `$ai-aura-start-sm` |
|                 | stop             | `$ai-aura-stop`     |
| Field           | border-bottom    | `$ai-border-strong` |
|                 | background color | `$field`\*          |
```

**Comment by aagonzales** [comment]
@alina-jacob I commented here the changes we were talking about in Slack. I only commented in the text input page. The number input will need similar updates.

File: src/pages/components/number-input/style.mdx [hunk 2]
```diff
@@ -6,6 +6,16 @@ description:
 tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 ---
 
+<AnchorLinks>
+
+<AnchorLink>Color</AnchorLink>
+<AnchorLink>Typography</AnchorLink>
+<AnchorLink>Structure</AnchorLink>
+<AnchorLink>Sizes</AnchorLink>
+<AnchorLink>AI Presenece</AnchorLink>
```
Comment (Kritvi-bhatia17) [inline src/pages/components/number-input/style.mdx:15]:
```suggestion
<AnchorLink>AI presence</AnchorLink>
```

File: src/pages/components/number-input/style.mdx [hunk 3]
```diff
@@ -174,3 +184,41 @@ layout, and design.
...
+
+[Learn more about the AI style elements]()
+
+| Element         | Property         | Token / Size      |
+| --------------- | ---------------- | ----------------- |
+| Linear gradient | start            | $ai-aura-start-sm |
+|                 | stop             | $ai-aura-stop     |
+| Field           | border-bottom    | $ai-border-strong |
+|                 | background color | $field\*          |
+| AI Slug         | size             | Mini              |
```
[the remark this answers is not in the document]
Comment (Kritvi-bhatia17) [reply src/pages/components/number-input/style.mdx:202]: A line will be added below the table 
image

File: src/pages/components/number-input/usage.mdx [hunk 4]
```diff
@@ -13,13 +13,23 @@ decrease the value with a two-segment control.
...
+</InlineNotification>
+
 <AnchorLinks>
 
 <AnchorLink>Overview</AnchorLink>
 <AnchorLink>Live demo</AnchorLink>
 <AnchorLink>Formatting</AnchorLink>
 <AnchorLink>Content</AnchorLink>
 <AnchorLink>Behaviors</AnchorLink>
+<AnchorLink>AI Presence</AnchorLink>
```
Comment (Kritvi-bhatia17) [inline src/pages/components/number-input/usage.mdx:32]:
```suggestion
<AnchorLink>AI presence</AnchorLink>
```

File: src/pages/components/number-input/style.mdx [hunk 5]
```diff
@@ -174,3 +184,41 @@ layout, and design.
 </div>
 
 <Caption>Sizes for number input fields | px / rem</Caption>
+
+## AI Presence
```
Comment (Kritvi-bhatia17) [inline src/pages/components/number-input/style.mdx:188]:
```suggestion
## AI presence
```

File: src/pages/components/text-input/style.mdx [hunk 6]
```diff
@@ -326,6 +327,44 @@ These sizes apply only to the default style of text input.
...
+| AI Slug         | size             | Mini              |
```
Comment (Kritvi-bhatia17) [inline src/pages/components/text-input/style.mdx:344]:
```suggestion
| AI Slug         | size             | mini              |
```

Comment (Kritvi-bhatia17) [reply src/pages/components/text-input/style.mdx:344]: A line will be added below the table 
image

File: src/pages/components/text-input/usage.mdx [hunk 7]
```diff
@@ -23,6 +34,7 @@ and short-form entries.
 <AnchorLink>Content</AnchorLink>
 <AnchorLink>Universal behaviors</AnchorLink>
 <AnchorLink>Modifiers</AnchorLink>
+<AnchorLink>AI Presenece</AnchorLink>
```
[the remark this answers is not in the document]
Comment (Kritvi-bhatia17) [reply src/pages/components/text-input/usage.mdx:37]:
```suggestion
<AnchorLink>AI presence</AnchorLink>
```

File: src/pages/components/text-input/usage.mdx [hunk 8]
```diff
@@ -543,6 +555,40 @@ met.
 </Column>
 </Row>
 
+## AI Presence
```
Comment (Kritvi-bhatia17) [inline src/pages/components/text-input/usage.mdx:558]:
```suggestion
## AI presence
```

File: src/pages/components/text-input/usage.mdx [hunk 9]
```diff
@@ -13,6 +13,17 @@ and short-form entries.
 
 </PageDescription>
 
+<InlineNotification>
+
+Experimental Slug is now available for
+[Text input](https://react.carbondesignsystem.com/?path=/story/experimental-unstable-slug-examples--text-input)
```
Comment (aagonzales) [inline src/pages/components/text-input/usage.mdx:19]: Lowercase `t`
```suggestion
[text input](https://react.carbondesignsystem.com/?path=/story/experimental-unstable-slug-examples--text-input)
```

File: src/pages/components/text-input/style.mdx [hunk 10]
```diff
@@ -326,6 +327,49 @@ These sizes apply only to the default style of text input.
 
 <Caption>Text input default style sizes | px / rem</Caption>
 
+## AI Presence
```
Comment (aagonzales) [inline src/pages/components/text-input/style.mdx:330]: Lowercase `p`
```suggestion
## AI presence
```

File: src/pages/components/text-input/style.mdx [hunk 11]
```diff
@@ -19,6 +19,7 @@ structure, and size.
 <AnchorLink>Typography</AnchorLink>
 <AnchorLink>Structure</AnchorLink>
 <AnchorLink>Size</AnchorLink>
+<AnchorLink>AI Presence</AnchorLink>
```
Comment (aagonzales) [inline src/pages/components/text-input/style.mdx:22]:
```suggestion
<AnchorLink>AI presence</AnchorLink>
```

File: src/pages/components/number-input/style.mdx [hunk 12]
```diff
@@ -174,3 +184,46 @@ layout, and design.
...
+the default variant.
+
+More information about AI style elements is coming soon.
+
+| Element         | Property         | Token / Size        |
+| --------------- | ---------------- | ------------------- |
+| Linear gradient | start            | `$ai-aura-start-sm` |
+|                 | stop             | `$ai-aura-stop`     |
+| Field           | border-bottom    | `$ai-border-strong` |
+|                 | background color | `$field\*`          |
```
Comment (aagonzales) [inline src/pages/components/number-input/style.mdx:201]:
```suggestion
|                 | background color | `$field`\*          |
```

**Review by aagonzales — COMMENTED** [review]
Looking good, just a few more formatting things.

**Review by guidari — APPROVED** [review]
LGTM! 🚀
Vercel for preview: https://carbondesignsystem-git-fork-alina-j-32ea40-carbon-design-system.vercel.app/

**Comment by Kritvi-bhatia17** [comment]
Hey @alina-jacob, everything else seems great with the changes/updates! ✨ 
Just a quick question: since number input includes both default and fluid states, should we display both states under the usage tab when AI is present? 🤔 
image

**Review by Kritvi-bhatia17 — COMMENTED** [review]
Hey @alina-jacob, everything else seems great with the changes/updates! ✨ 
Just a quick question: since number input & text input include both default and fluid states, should we display both states under the usage tab under 'AI presence' section? 🤔 
image

**Comment by alina-jacob** [comment]
> Hey @alina-jacob, everything else seems great with the changes/updates! ✨ Just a quick question: since number input & text input include both default and fluid states, should we display both states under the usage tab under 'AI presence' section? 🤔 image


The [guideline doc](https://www.figma.com/file/OJwBWBlctEgm9zYUCYbhu3/Carbon-for-AI-%E2%80%93-website-integration?type=design&node-id=80%3A6998&mode=design&t=K4jVLRCSy9JvTCML-1) showed just default so went with that, should we be showing default and fluid states for the **usage tab** as well @aagonzales?

**Comment by aagonzales** [comment]
@alina-jacob I believe we discussed this in Slack, but we decided to include both the default and fluid tabs. Part of the work is also to now avoid the word "default" in the copy text. It should be changed to "non-AI" instead of "default"

**Review by Kritvi-bhatia17 — APPROVED** [review]
LGTM! 🎉

**Review by aagonzales — APPROVED** [review]
Looks good! Ship it!

**Comment by Kritvi-bhatia17** [comment]
Hi @alina-jacob, just a small note - you might need to add the page description on the Number Input STYLE tab, similar to the Text Input.
image
image

**Pull request merged**